### PR TITLE
New package: sigoden.Dufs version 0.34.1

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92153)

### CODE REVIEW

**Comment by mdanish-kh** [comment]
@msftbot .zip

**Review by ItzLevvie — DISMISSED** [review]

**Comment by SpecterShell** [comment]
Hold

File: manifests/s/sigoden/Dufs/0.31.0/sigoden.Dufs.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: sigoden.Dufs
+PackageVersion: 0.31.0
+InstallerType: zip
+ReleaseDate: 2022-11-12
+Installers:
```
Comment (stephengillie) [inline manifests/s/sigoden/Dufs/0.31.0/sigoden.Dufs.installer.yaml:8]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Comment (stephengillie) [reply manifests/s/sigoden/Dufs/0.31.0/sigoden.Dufs.installer.yaml:8]: Hi @SpecterShell,

The package installs, but gives this message when launched:
image

File: manifests/s/sigoden/Dufs/0.31.0/sigoden.Dufs.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: sigoden.Dufs
+PackageVersion: 0.31.0
+InstallerType: zip
+ReleaseDate: 2022-11-12
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+Installers:
```
Comment (stephengillie) [inline manifests/s/sigoden/Dufs/0.31.0/sigoden.Dufs.installer.yaml:11]:
```suggestion
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Comment (stephengillie) [reply manifests/s/sigoden/Dufs/0.31.0/sigoden.Dufs.installer.yaml:11]: Apparently, I added a bad line ending or other invisible character with my previous suggestion. I believe that I have corrected it in this suggestion. Thank you for your patience.

Comment (SpecterShell) [reply manifests/s/sigoden/Dufs/0.31.0/sigoden.Dufs.installer.yaml:11]: image
And nothing wrong here. 🤔

**Comment by mdanish-kh** [comment]
[Policy] reset feedback

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by stephengillie** [comment]
With the dependency preinstalled, this package installs and launches normally.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm seeing this:
```
Enter a pull request number: 92153

Manifest validation succeeded.
Found Dufs [sigoden.Dufs] Version 0.34.1
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/sigoden/dufs/releases/download/v0.34.1/dufs-v0.34.1-x86_64-pc-windows-msvc.zip
  ██████████████████████████████  1.33 MB / 1.33 MB
Successfully verified installer hash
An anti-virus product reports an infection in the installer

Press any key to continue . . .
```

Tested this on a Hyper-V VM with latest Defender definitions as of 11th of June 2023:
image
image
image

with `winget settings --enable LocalArchiveMalwareScanOverride` and `--ignore-local-archive-malware-scan --accept-package-agreements` passed to `winget install --manifest`

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
> I'm seeing this:
> ```
> An anti-virus product reports an infection in the installer
> 
> Press any key to continue . . .
> ```
> 

Found in a newer version of Defender (1144) vs the version available on 6/2 (298):  
> 6:32:49 PM: Detected 0 Defender (with signature version 1.391.298.0) errors: 
[sigoden.Dufs.x86.0.34.0.log](https://github.com/microsoft/winget-pkgs/files/11725296/sigoden.Dufs.x86.0.34.0.log)

**Pull request closed without merging**